oi/OITraceCode.cpp

```diff
@@ -20,7 +20,7 @@
 #define C10_USING_CUSTOM_GENERATED_MACROS
 
 // These globals are set by oid, see end of OIDebugger::compileCode()
-extern uintptr_t dataBase;
+extern uint8_t* dataBase;
```

Comment:
The types of this variable seem all kinds of crazy in FuncGen...
Does changing its type here help with something?